src/braket/aws/aws_quantum_task_batch.py

```diff
@@ -98,15 +102,16 @@ def _execute(
         s3_destination_folder,
         shots,
         max_parallel,
+        max_workers,
         poll_interval_seconds,
         *args,
         **kwargs,
     ):
-        # These two lists track the number of remaining and currently executing tasks
-        # Since appending and popping are atomic, they serve as locks
+        # Tracks the number of remaining and currently executing tasks
+        # Since appending and popping are atomic, it serve as a lock too
```

Comment:
# Because appending and popping are atomic, it serves as a lock, too

(Note to writer: It had a typo, it needed a coma, and you really meant "because" as in causality, not "since" as in a period of time has elapsed)

Reply:
needed a comma, not a coma, hahah!